src/main/java/com/softserve/kickscooter/kickscootervehicle/management/controller/KafkaController.java

```diff
@@ -0,0 +1,51 @@
+package com.softserve.kickscooter.kickscootervehicle.management.controller;
+
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterRawDataDto;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterStatusDto;
+import com.softserve.kickscooter.kickscootervehicle.management.model.ScooterStatus;
+import com.softserve.kickscooter.kickscootervehicle.management.service.StatusService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.geo.Point;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+@AllArgsConstructor
+@Slf4j
+public class KafkaController {
+
+    private static final String statusTopic = "all_scooter_data_topic";
```

Comment:
1. Constants name in java should have UPPER_SNAKE_CASE.
2. What is the purpose of these constants? Their name is almost the same as the topic name. 
3. Name topics similarly, either snake_case or kebab-case

Reply:
I use them in logging. Names fixed